server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ErrorCodes.java

```diff
@@ -225,6 +225,8 @@ public enum ErrorCodes implements ErrorCode {
     ObGetExecuteDetailFailed,
     ObGetPlanExplainFailed,
     ObGetPlanExplainEmpty,
+    ObGetFullLinkTraceFailed,
+    FullLinkTraceNotAvailable,
```

Comment:
ObFullLinkTraceNotSupported 

not available seems service not working, 'not supported' will be an expected behavior.

Reply:
Very reasonable. Done.